### DISCUSSION TITLE
Adopters

### DISCUSSION THREAD

**Discussion by musingvirtual** [open-telemetry/opentelemetry.io#4299] [Content Refactoring]
Raising an issue about [this page](https://opentelemetry.io/ecosystem/adopters/) as per [Slack conversation](https://cloud-native.slack.com/archives/C02UN96HZH6/p1709852140493519):

We've had an end user asking for: "Is there any stats, references, etc around OTel industry adoption? I've heard its one of the most active CNCF projects, which is incredible. Curious if there's any other data points that I can arm myself with to show momentum and "this is the way" within my company."

The handful of company names listed here are not really representative of the breadth of adoption and do not make us look like the industry standard.  While I appreciate these lists are hard to keep up to date and that we are hopeful we'll soon figure out other ways to get information about adoption, it would be immediately helpful for folks trying to make a business case for adoption if we could get this list up to something like 50 adopters with some more recognizable company names.

I propose that we divide this into two sections so that it's easier for the EUWG to go out and encourage more end users to make pull requests, and adjust the blurb at the bottom about who is eligible for this page accordingly - 

1) featured adopters at the top who have blog posts, case studies, and other collateral to share (in some orgs folks may need permission to be featured in this way and they may also need to write such collateral).
2) adopters who just want us to include their company name (a simple PR that most people can make).

We also need to decide what to do with vendors who are dogfooding.  My personal opinion about this is that we have another list for [vendors](https://opentelemetry.io/ecosystem/vendors/), but might want to consider allowing some larger companies that have an observability product plus a non-observability product on both lists case by case.

I will also point out that prometheus.io has a big pile of logos of both end users and vendors right on their homepage - similar to how most SaaS software has such logos on the homepage - and this might be worth considering for future for opentelemetry.io.

**Comment by svrnm** [comment]
Thanks, as discussed on slack, I am in support of doing that split.

We also talked about the same yesterday during the SIG call, a few additional points came up:

* for the dogfooding vendors we can add some tagging (an asterisk letting people know that this is a vendor, etc.) or keeping that requirement up that vendors need to point to their dog fooding blog post, which they very likely have
* we might also need to tag "end users by the definition of CNCF", e.g. vendors might not fit that description in that case and a few others as well, but that's a concern for later.
* we also need to drop that note on removing adopters without a reference by earlier this year (I take all blame for setting this up...)

On the logos: I am happy to have them, but IANAL, so I don't know what we need to do to use them on our website, since those logos are probably conflicting with our CC BY license.
